[2025-05-06T03:58:18.046+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-06T03:58:18.068+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: BuildELT_dbt.dbt_test manual__2025-05-06T03:57:53.262913+00:00 [queued]>
[2025-05-06T03:58:18.079+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: BuildELT_dbt.dbt_test manual__2025-05-06T03:57:53.262913+00:00 [queued]>
[2025-05-06T03:58:18.081+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-05-06T03:58:18.096+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): dbt_test> on 2025-05-06 03:57:53.262913+00:00
[2025-05-06T03:58:18.103+0000] {standard_task_runner.py:72} INFO - Started process 3088 to run task
[2025-05-06T03:58:18.106+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'BuildELT_dbt', 'dbt_test', 'manual__2025-05-06T03:57:53.262913+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/build_elt_with_dbt.py', '--cfg-path', '/tmp/tmp0h_0ae2b']
[2025-05-06T03:58:18.109+0000] {standard_task_runner.py:105} INFO - Job 36: Subtask dbt_test
[2025-05-06T03:58:18.151+0000] {task_command.py:467} INFO - Running <TaskInstance: BuildELT_dbt.dbt_test manual__2025-05-06T03:57:53.262913+00:00 [running]> on host 4bf74f6ca137
[2025-05-06T03:58:18.254+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='BuildELT_dbt' AIRFLOW_CTX_TASK_ID='dbt_test' AIRFLOW_CTX_EXECUTION_DATE='2025-05-06T03:57:53.262913+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-06T03:57:53.262913+00:00'
[2025-05-06T03:58:18.256+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-06T03:58:18.271+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-06T03:58:18.275+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/***/.local/bin/dbt test --profiles-dir /opt/***/tesla_project --project-dir /opt/***/tesla_project']
[2025-05-06T03:58:18.291+0000] {subprocess.py:86} INFO - Output:
[2025-05-06T03:58:20.639+0000] {subprocess.py:93} INFO - [0m03:58:20  Running with dbt=1.8.7
[2025-05-06T03:58:21.646+0000] {subprocess.py:93} INFO - [0m03:58:21  Registered adapter: snowflake=1.8.1
[2025-05-06T03:58:22.075+0000] {subprocess.py:93} INFO - [0m03:58:22  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-05-06T03:58:22.076+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2025-05-06T03:58:22.077+0000] {subprocess.py:93} INFO - - models.tesla_project.example
[2025-05-06T03:58:22.285+0000] {subprocess.py:93} INFO - [0m03:58:22  Found 3 models, 1 snapshot, 1 source, 459 macros
[2025-05-06T03:58:22.289+0000] {subprocess.py:93} INFO - [0m03:58:22
[2025-05-06T03:58:22.290+0000] {subprocess.py:93} INFO - [0m03:58:22  Nothing to do. Try checking your model configs and model specification args
[2025-05-06T03:58:23.164+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-05-06T03:58:23.187+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-06T03:58:23.189+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=BuildELT_dbt, task_id=dbt_test, run_id=manual__2025-05-06T03:57:53.262913+00:00, execution_date=20250506T035753, start_date=20250506T035818, end_date=20250506T035823
[2025-05-06T03:58:23.230+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-06T03:58:23.255+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-06T03:58:23.259+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
